hw/top_earlgrey/ip/ast/data/ast.hjson

```diff
@@ -36,42 +30,459 @@
   ],
   regwidth: "32",
   registers: [
-    { multireg:
-      {
-        name: "REGA",
-        desc: "AST Registers Array-A for OTP/ROM Write Testing",
-        count: "NumRegsA",
-        cname: "REGA",
-        swaccess: "rw",
-        hwaccess: "hro",
-        fields: [
-          { bits: "31:0",
-            name: "reg32",
-            desc: "32-bit Register",
-            resval: "0",
-          },
-        ],
-      },
+    { name: "REGA0",
```

Comment:
hmm... i wonder if in the future it makes sense to allow for slight tweaks in multi-reg, ie they don't all have to be identical in attributes.  
@eunchan 
@msfschaffner 

what do you guys think? I think it boils down to what we think of mult-reg.
Are they a convenient array where we compact? or is it a duplication of a single register?

Allowing attribute changes could make things very complicated.. so i'm leaning towards no.

Reply:
I see no real value in that. If you need different attributes for each register, you better define each register separately.

For the real AST, all registers are external due to security features and as a result it breaks the automated RAL model that need to be fixed with special function (tried to share it...).